pyne/mcnp.py

```diff
@@ -793,7 +793,7 @@ def read(self):
             # Handle continuation lines
             while words[-1] == '+':
                 extraWords = self.f.readline().split()
-                words = words + extraWords
+                words = words[:-1] + extraWords  # Correct bug in the XSDIR parsing
```

Comment:
```suggestion
                words = words[:-1] + extraWords
```
This was originally suggested by @pshriwise - but I'll add it here to expidite.